Add ``osu! Philippines Nationals 2024 - osu!taiko`` tournament article (#12637)
ppy/osu-wiki: wiki/Tournaments/oPHN/2024_taiko/en.md

---
tags:
  - PHNT2024
  - PHNT 2024
---

# osu! Philippines Nationals 2024 - osu!taiko

![PHNT2024 logo](img/logo.jpg)

> The **osu! Philippines Nationals 2024 - osu!taiko** Tournament (***PHNT2024***) was a 1v1 rank-restricted osu!taiko tournament. It serves as the National Championships of osu!taiko for the Philippine community of the game.

Reviewer: ```suggestion
The **osu! Philippines Nationals 2024 - osu!taiko** Tournament (***PHNT2024***) was a 1v1 rank-restricted osu!taiko tournament. It served as the National Championships of osu!taiko for the Philippine community of the game.
```

> ## Tournament Schedule

Reviewer: ```suggestion
## Tournament schedule
```

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-09-01/2024-09-01 (15:59 UTC) |
| Qualifiers showcase | 2024-10-06 (13:00 UTC) |
| Qualifiers | 2024-10-12/2024-10-13 |
| Round of 16 | 2024-10-19/2024-10-20 |
| Quarterfinals | 2024-10-26/2024-10-27 |
| Semifinals | 2024-11-02/2024-11-03 |
| Finals | 2024-11-09/2024-11-10 |
| Grand Finals | 2024-11-16/2024-11-17 |

## Prizes

The winner of the tournament will receive a profile badge.

![](img/phnt-badge.png "PHNT winner badge")

> ## Organization

Reviewer: ```suggestion
## Organisation
```

> The osu! Philippines Nationals 2024 - osu!taiko tournament is ran by various community members.

Reviewer: ```suggestion
The osu! Philippines Nationals 2024 - osu!taiko tournament was ran by various community members.
```

| GFX | ::{ flag=PH }:: [muffleberry](https://osu.ppy.sh/users/8978259), ::{ flag=PH }:: [Miyazumi](https://osu.ppy.sh/users/9919550), ::{ flag=PH }:: [yukina meng](https://osu.ppy.sh/users/9919550) |
| Statistician | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1971304)
- Livestream
  - [Twitch](https://www.twitch.tv/osuliveph)
  - [Twitch B-stream](https://www.twitch.tv/osuliveph_2)
  - [YouTube](https://www.youtube.com/channel/UCkV4V1OVO6gOGxtH51glH0Q)
- [VOD Archive](https://www.youtube.com/playlist?list=PLiCZoouucYhCuC2wW_bBXoT0pTCC76YN5)
- [PHNT Discord server](https://discord.gg/KFn8rBxf42)

> - [Challonge](https://challonge.com/PHNT2024)

Reviewer: ```suggestion
- [Challonge bracket](https://challonge.com/PHNT2024)
```

| 8 | ::{ flag=PH }:: [wwowie](https://osu.ppy.sh/users19445999) |
| 9 | ::{ flag=PH }:: [Asa Kazato](https://osu.ppy.sh/users13547767) |
| 10 | ::{ flag=PH }:: [Kornettissimo](https://osu.ppy.sh/users7487058) |
| 11 | ::{ flag=PH }:: [Syni-](https://osu.ppy.sh/users15232470) |
| 12 | ::{ flag=PH }:: [Euanofthedead](https://osu.ppy.sh/users15676812) |
| 13 | ::{ flag=PH }:: [\_Kancho\_](https://osu.ppy.sh/users11035895) |
| 14 | ::{ flag=PH }:: [Kotaro](https://osu.ppy.sh/users796530) |
| 15 | ::{ flag=SA }:: [thesuslulkid](https://osu.ppy.sh/users25611653) |
| 16 | ::{ flag=PH }:: [Phann\_](https://osu.ppy.sh/users12905445) |

## Podium

This competition ended with the following podium:

| Placing | User |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [Silicosis 2](https://osu.ppy.sh/users18560307) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [Astrid Park](https://osu.ppy.sh/users12320451) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [Taikore](https://osu.ppy.sh/users29115112) |

## Mappools

### Grand Finals

> **[You may download the mappack here](https://drive.google.com/file/d/1YpWSArtywv5Q6cD2gkHah9tMTqFyRG7Z/view?usp=sharing)**

Author: should be updated to follow this now

> - NoMod

Reviewer: ```suggestion
- No Mod
```

  2. [Sakuzyo feat. Nikki Simmons - Destr0yer (Fynbi) \[0ni\]](https://osu.ppy.sh/beatmapsets/2278469#taiko/4856321)
- Hard Rock
  1. [Dictate - IMITATE (Defectum) \[ky cuc (PHNT Edit)\]](https://osu.ppy.sh/beatmapsets/2226955#taiko/4724288)
  2. [HARI KARI - MIRROR FRAGMENT (D3kuu) \[INNER ONI\]](https://osu.ppy.sh/beatmapsets/971297#taiko/2034220)
- Double Time
  1. [Yunomi - Game Over (feat. TORIENA) (komasy) \[Game Start\]](https://osu.ppy.sh/beatmapsets/949517#taiko/1982765)
  2. [Halozy - Paranoid Lost (Kin) \[Autumn\]](https://osu.ppy.sh/beatmapsets/683947#taiko/1447507)
- Forced Mod
  1. [USAO - Knight Rider (midorijeon) \[INFERNO\]](https://osu.ppy.sh/beatmapsets/2197626#taiko/4650522)
  2. [BlackY - ARTEMiS (ikin5050) \[Dusk's Inner Oni\]](https://osu.ppy.sh/beatmapsets/1314196#taiko/3393312)
  3. [Feryquitous - Rhuzerv (Fynbi) \[Voice\]](https://osu.ppy.sh/beatmapsets/2278462#taiko/4856312)
- Tiebreaker
  1. [Akiri - The Angel Of Fallen Origins (qish) \[Grand Finale\]](https://osu.ppy.sh/beatmapsets/2279014#taiko/4857681)

### Finals

**[You may download the mappack here](https://drive.google.com/file/d/10ITDqA0g4CqBhZMzBvhsLrRzKrdWw2rz/view?usp=sharing)**

> - NoMod

Reviewer: ```suggestion
- No Mod
```

  2. [Neko Hacker - Pictures feat. 4s4ki (Irone OSU) \[Capture this dawn into my heart\]](https://osu.ppy.sh/beatmapsets/1644680#taiko/3357161)
- Hard Rock
  1. [Camellia feat. Nanahira - POLKAMANIA (Faputa) \[INNER ONI\]](https://osu.ppy.sh/beatmapsets/1613520#taiko/3297906)
  2. [Lon - Yuru Fuwa Jukai Girl (Catbug1524) \[Carefree\]](https://osu.ppy.sh/beatmapsets/1220631#taiko/2539220)
- Double Time
  1. [PinocchioP - Thanks for being Lifeless (SuperJSpectrixV) \[Lifeless Oni\]](https://osu.ppy.sh/beatmapsets/891091#taiko/1862716)
  2. [kamome sano - starlights feat. TEA (you Remix) (ler1211) \[nokduro's oni\]](https://osu.ppy.sh/beatmapsets/1778120#taiko/3641591)
- Forced Mod
  1. [xi - Ascension to Heaven (Shiirn) \[Ono's Taiko Oni\]](https://osu.ppy.sh/beatmapsets/34348#taiko/119666)
  2. [32ki feat. Hatsune Miku & Kasane Teto - Mesmerizer (HiroK) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/2191205#taiko/4634769)
  3. [Irodorimidori - Shikisai Kajou no Diary Music (\_gt) \[Colorful Days\]](https://osu.ppy.sh/beatmapsets/2275130#taiko/4848014)
- Tiebreaker

>   1. [Arctic Indexed Systems. - Nightburner (Eyenine) \[Shadeblight\]](https://osu.ppy.sh/beatmapsets/2275218#taiko/4848179)

Reviewer 2: all tiebreakers should be bolded, not just the first one listed

| **Taikore** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: Kotaro | *win by default* |
| **Krishel17** ::{ flag=PH }:: | **5** | 2 | ::{ flag=PH }:: Euanofthedead | [#1](https://osu.ppy.sh/community/matches/115914838) |
| Farmer Brown ::{ flag=PH }:: | 3 | **5** | ::{ flag=PH }:: **Syni-** | [#1](https://osu.ppy.sh/community/matches/115914965) |
| **Astrid Park** ::{ flag=PH }:: | **5** | 0 | ::{ flag=SA }:: thesuslulkid | [#1](https://osu.ppy.sh/community/matches/115915618) |

### Qualifiers

The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1WF-I_EkpFGLJmB-19lLCu-OpBSiXvUFj6l3QrQrsraU/edit?rm=minimal)

## Ruleset

### General Rules

- Players must be in the rank range of #500 to #10,000 to be eligible to join.
- Players who currently have (or had at some point since account creation) the Philippine Flag on their profile are the only ones allowed to join the tournament.
- Score v2 will be used.
- Permitted mods for Forced Mod (FM) and Tiebreaker picks are Hidden (HD), HardRock (HR), or both.
- No Fail (NF) is enforced for all picks.

### Round Overview

#### Qualifier Stage

> - Players must sign up to one of the qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time.
> - There will be no warmups for all qualifiers lobbies. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1, DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.

Reviewer 2: ```suggestion
- Players must sign up to one of the Qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifier mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time.
- There will be no warmups for all Qualifier lobbies. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1, DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.
```

- The higher scores from each player for each map will be used for seeding. Players will be ordered by overall rank, which is computed by taking the average of their rankings on each map. In case of a tie, the sum of the top scores of the tied players are taken into account instead. Sharing of scores before the results showcase is discouraged.
- The number of players that will advance to the next stage will depend on the total number of participants.

#### Bracket Stage

> - Players are placed into a double-elimination bracket. They comprise of the Top 16 from the qualifiers.
> - Mappool Sizes and Best of Format for each round are given as follows:

Reviewer 2: ```suggestion
- Players are placed into a double-elimination bracket. They comprise of the top 16 from the Qualifiers.
- Mappool sizes and best-of format for each round are given as follows:
```

  - Round of 16 (Best of 9): 4 NM, 2 HD/HR/DT, 3 FM, 1 TB
  - Quarterfinals and Semifinals (Best of 11): 5 NM, 2 HD/HR/DT, 3 FM, 1 TB
  - Finals and Grand Finals (Best of 13): 6 NM, 2 HD/HR/DT, 3 FM, 1 TB

### Match Rules

> - The multiplayer room will be created and invites sent to players 5-10 minutes prior to the match start time. The multiplayer rooms are named in the following format: `PHNT2024: (<Player 1>) vs. (<Player 2>)`

Reviewer 2: we tend to use curly braces for placeholders

```suggestion
- The multiplayer room will be created and invites sent to players 5–10 minutes prior to the match start time. The multiplayer rooms are named in the following format: `PHNT2024: ({Player 1}) vs. ({Player 2})`
```

> - If a player is more than 10 minutes late to the for their match, the player inside the MP room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.

Reviewer 2: ```suggestion
- If a player is more than 10 minutes late to their match, the player inside the multiplayer room will be declared the winner. If neither player appears, the player with the higher seed will be considered the winner.
```

> - Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use DoubleTime if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.

Reviewer 2: ```suggestion
- Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use Double Time if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player's warm-up is skipped.
```

- Players are required to ban 1 map in the pool, excluding the tiebreaker. Banned charts cannot be picked during the match proper. The players shall use the `!roll` command in-game to determine the order in which charts are banned and picked. The order is as follows:

>   - Player with the higher roll: First Pick
>   - Player with the lower roll: First Ban

Reviewer 2: ```suggestion
  - Player with the higher roll: first pick
  - Player with the lower roll: first ban
```

  - The ban phase happens first, then the match proceeds to the picks.

> - Each player will alternate picks based on the `!roll` results until a winner is determined based on the Best-of format for the round. If both players are tied on match point (e.g. 4-4 score in a Best-of-9), the players will play the tiebreaker map of the pool to determine the winner.
> - Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random in behalf of the player.
> - Players are eligible to use a 2 minute tactical time out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.

Reviewer 2: ```suggestion
- Each player will alternate picks based on the `!roll` results until a winner is determined based on the best-of format for the round. If both players are tied on match point (e.g. 4–4 score in a best-of-9), the players will play the tiebreaker map of the pool to determine the winner.
- Players are given 2 minutes to ban/pick maps when it is their turn. Failure to do so means that the referee will ban/pick at random on behalf of the player.
- Players are eligible to use a 2-minute tactical time-out once per match. They are only allowed to use it when it is their turn to pick. It will take into effect after the pick timer has expired.
```

- If a player disconnects during a map within 30 seconds, the player must reconnect within 5 minutes and the map may be replayed. This only applies once for each player. From there onwards, if the player disconnects again at any point of the match, the match will be played out normally until the player reconnects or forfeits the remaining maps.
- If two players tie on a map, the map is replayed. In case another tie occurs, the map is nullified and another map may be chosen by the same player who picked the nulled map.

> - Scores in the match link are final, unless a player can give a legitimate screenshot (Shift + F12), a replay file, or a video clip. The score in the given proof must be clearly visible in order for it to be accepted.

Reviewer 2: ```suggestion
- Scores in the match link are final, unless a player can give a legitimate screenshot (`Shift` + `F12`), a replay file, or a video clip. The score in the given proof must be clearly visible in order for it to be accepted.
```
